src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -127,7 +130,7 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
     }
 
     get isEditMode() {
-        return this.recordId ? true : false;
+        return !!this.recordId;
```

Comment:
Would this be the same as` return this.recordId `?

Reply:
Yes, maybe this one is better as `return isNotBlank(this.recordId);`